### PR TITLE
fix(watchman)

### PR DESCRIPTION
**Pull request by Dunedubby**

### CODE REVIEW

**Comment by Dunedubby** [comment]
Judging from the logs, we may need to update folly to a more recent version. The correct version (2024.09.09.00) has not yet been released on pkgx.

**Comment by jhheider** [comment]
yeah, they update every week, but have to be built in a specific order. i'll get them finished tomorrow. though watchman hasn't built for our linux targets in a few months.

**Comment by Dunedubby** [comment]
Gotcha. Thanks so much for maintaining these. By the way, what's the best
way to set the build-time WATCHMAN_STATE_DIR so it points to the .pkgx
directory?

**Comment by jhheider** [comment]
that should probably be an `runtime.env.WATCHMAM_STATE_DIR: ${{prefix}}/var` unless there's a reason we shouldn't hijack it.

File: projects/facebook.com/watchman/package.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ build:
     WATCHMAN_STATE_DIR: "{{prefix}}/var/run/watchman"
     CMAKE_ARGS:
       - -DCMAKE_INSTALL_PREFIX="{{prefix}}
-      - -DCMAKE_PREFIX_PATH=$(echo $LIBRARY_PATH | sed 's/\/lib:/:/g; s/\/lib$//')
+      - -DCMAKE_PREFIX_PATH=$(echo $LIBRARY_PATH | sed 's/\/lib:/;/g; s/\/lib$//')
```
Comment (jhheider) [inline projects/facebook.com/watchman/package.yml]: CMAKE_PREFIX_PATH should be fixed in pkgx 1.2.1 again. sorry for the breakage!

Comment (Dunedubby) [reply projects/facebook.com/watchman/package.yml]: Ah okay haha I was wondering what was happening...

**Pull request closed without merging**